composer.json

```diff
@@ -14,7 +14,7 @@
     "php": "^7.1",
     "guzzlehttp/guzzle": "~6.0 | ~7.0",
     "ext-json": "*",
-    "lcobucci/jwt": "^3.3",
+    "lcobucci/jwt": "^3.3 | ^4.0.0",
```

Comment:
4.0 of this library is a major interface refactoring that is not compatible simply as a drop in replacement for 3.3, it will break the library in a number of ways

Reply:
working on it ;-)

Reply:
Sounds good. Just a heads up, that work should probably be captured in a separate PR so we can address things separately if it isn't directly related to this hotfix